### ISSUE TITLE
Is it possible to release the script for running in the wild or atleast can you release the CPN model fine tuned on Human3.6M?

### ISSUE THREAD

**Issue by abhikhanna30** [facebookresearch/VideoPose3D#12]

**Comment by dariopavllo** [comment]
As I said [here](https://github.com/facebookresearch/VideoPose3D/issues/2#issuecomment-446026483), the purpose of this repo is to reproduce the results of the paper. We added the YouTube videos to show more qualitative results, but we performed the whole process manually since it involves changes to multiple codebases.

We may release some scripts in the future to perform this manually, but we don't have a precise timeline.

Regarding the CPN model, for the videos in the wild we used the pretrained one (i.e. we didn't fine-tune on H3.6M).

**Comment by abhikhanna30** [comment]
Okay so if you did not fine tune it , wont the CPN model produce COCO keypoints? Because you have mentioned in previous issues you used this model(
https://s3.amazonaws.com/video-pose-3d/pretrained_h36m_cpn.bin) for when the outputs are from CPN.

**Comment by dariopavllo** [comment]
Good point, we haven't released the model trained on CPN COCO keypoints.

I'll try to make it available soon.

**Comment by dariopavllo** [comment]
Hi,

Here is the link to the model: https://s3.amazonaws.com/video-pose-3d/cpn-pt-243.bin
This is the one we used for generating the videos in the wild. It takes COCO keypoints from CPN as input, and outputs Human3.6M joints. The model learns to map the different sets of keypoints.

For CPN, you should use a pretrained model based on the ResNet50 architecture with 384x288 resolution. [This](https://github.com/GengDavid/pytorch-cpn) is a very good open-source implementation for PyTorch.

**Comment by abhikhanna30** [comment]
Hi Dario,
Thanks for providing that.
I just one last doubt. Now according to you , this CPN-FT inputs coco points and outputs H36M points.
So in the paper ,what is the difference between D-PT and CPN-FT?
